server/routes/api/modules.js

```diff
@@ -5,8 +5,16 @@ import validator from '../../middleware/validations/test';
 
 const router = express.Router();
 
-router.get('/modules', moduleController.getAllModules);
-router.get('/modules/:id', moduleController.getModule);
+router.get(
+  '/modules',
+  Authenticate.verifyToken,
```

Comment:
I think this verifyToken middleware already tells whether the user is logged in or not

Reply:
Yes,  it does.